### ISSUE TITLE
Caret height when using Custom Segment

### ISSUE THREAD

**Issue by mambastudio** [FXMisc/RichTextFX#1026]
Hi,

Thank you for this great library. It has been quite a blessing using it. 

I needed some information, and possibly a solution to a problem, concerning an aesthetic. Apparently, when one uses a custom segment, let's say an image, which has different dimensions that are larger than the current row height before insertion, the caret scales to the height of the image. As illustrated in the diagram (caret not visible but the drawing illustrates the issue). 

![image](https://user-images.githubusercontent.com/3126668/123781261-b7069a00-d8dc-11eb-914d-20800d61b6f3.png)

Is there a possibility of ensuring the caret height remain same height as before without adjusting height. I've searched through the Issues (open and closed), but couldn't find a similar concern. This can be replicated by the current Rich Text Editor demo, which the program has been modified from. Currently using RichText 0.10.5 (with jdk 1.8)

**Comment by Jugen** [comment]
Sorry, I don't think there is. I've also seen this behavior myself and also thought it was ugly :-)
I suppose some sort of test could be added (not sure about that) so that when the previous segment isn't Text then the caret doesn't grow/adjust .... ?

**Comment by mambastudio** [comment]
Yes, if there is a way that can be done, that would be awesome. But if not possible now, I can try to live with it and get used to it. Lol.
